### DISCUSSION TITLE
Some suggestions for improving the UI in the Elementor editor

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#22137] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

In version 3.13, the favorites widgets section was removed, to display conditionally. It was a great decision for that.

In the following I have a few more requests to improve the widgets panel, I hope it will be useful, because I personally think it will be very useful.

1- Remove the global widgets section from the current location and display it conditionally like the favorites widgets.

![a](https://user-images.githubusercontent.com/88034902/233429263-ffde56fc-70dd-4908-8f55-4a2d0d7011dc.png)

2- If you remove the global widgets section from the current location as suggested above, two more sections will remain. 1- Elements and 2- Widgets. **In this case, both of them will not really be needed. Because the panel will be completely dedicated to widgets.**

**(Please note that deleting these two texts does not compromise anything. Because the widgets panel is completely clear. Whether these two texts exist or not)**

![editor (now)](https://user-images.githubusercontent.com/88034902/233430070-2bf48e4f-81b7-4492-9ace-1f60af1c9c96.png)

3- If you delete the 3 sections mentioned above, make the search box of the widgets panel a little bigger.

### Describe the solution you'd like

In general, the above 3 suggestions will make the widget panel much more optimized than it is now. For example, when we don't save any widgets as global widgets, we won't see the global widgets section. Immediately after adding the widget as a global widget, this section will be added in the widgets panel. Just like saving widgets as favorites in version 3.13.0

### Considering the above, the widgets panel will be as shown below.

![editor](https://user-images.githubusercontent.com/88034902/233431076-5889788b-7751-47af-9afd-f7a5081ac85f.png)

I hope it was useful.

**Comment by fabigrafo** [comment]
I think or hope that Elementor will completely redesign this sidebar anyway. Your tips are useful and I suspect Elementor will implement similar things. It would also be cool if the sidebar was detachable.

**Comment by 79ho3ein** [comment]
Great ideas👌

**Comment by nicholaszein** [comment]
Hi @Michaelgimii,

thank you for the suggestions! 🙏 

While we might change when and how the panel title appears in the future (we are still discussing this), moving Global widgets to a section instead of leaving them in their own tab is not something we will do, simply because you can have infinite Global widgets, and they need their own space. If we add them as a section, they would completely overtake the interface, or we would need to place them all the way to the end of the list, which would make working with them a terrible chore.

We will consider the suggestions here, but the Global widgets, have their own tabs because, from a UX perspective, this makes navigation easy and clutter-free.

Kind regards
